### ISSUE TITLE
NameError: global name 'args' is not defined

### ISSUE THREAD

**Issue by xiconet** [Janhouse/tespeed#30]
Testing the script without argument:

````
C:\Documents and Settings\arc\Mes documents\development\python>tespeed
Getting ready. Use parameter -h or --help to see available features.
Loading speedtest configuration...
IP: 83.153.78.56; Lat: 42.697600; Lon: 2.895400; ISP: Free SAS
Loading server list...
Looking for closest and best server...
Testing latency...
52 ms latency for http://perpignan.old-linux.com/speedtest/ (Old-Linux.com, Perpignan, France) [0.11 km]
105 ms latency for http://speedtest.prisconetworks.com/speedtest/ (Prisco Networks, L'Escala, Spain) [68.78 km]
77 ms latency for http://speedtest.aerisnavigo.com/speedtest/ (Aeris Navigo, Girona, Spain) [79.59 km]
112 ms latency for http://speedtest.altanetica.es/speedtest/ (Altanetica SL, Calonge, Spain) [94.83 km]
70 ms latency for http://speedtest-mpl.neotelecoms.com/speedtest/ (NEOTELECOMS,
Montpellier, France) [148.96 km]
Process Process-2:
Traceback (most recent call last):
  File "C:\Python27\lib\multiprocessing\process.py", line 258, in _bootstrap
    self.run()
  File "C:\Python27\lib\multiprocessing\process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Documents and Settings\arc\Mes documents\development\python\tespeed.py", line 328, in AsyncGet
    print_debug('
                            \r')
  File "C:\Documents and Settings\arc\Mes documents\development\python\tespeed.py", line 604, in print_debug
    if args.suppress != True:
NameError: global name 'args' is not defined
````
I really don't understand this error. Please help a poor "windoze" user if you can.

**Comment by xiconet** [comment]
no fix

**Comment by Janhouse** [comment]
Should be fixed in  efbf2d3 now.
